core/src/rpc_pubsub_service.rs

```diff
@@ -16,39 +16,81 @@ use std::{
     time::Duration,
 };
 
+#[derive(Debug, Clone)]
+pub struct PubSubConfig {
+    // See the corresponding fields in
+    // https://github.com/paritytech/ws-rs/blob/be4d47575bae55c60d9f51b47480d355492a94fc/src/lib.rs#L131
+    // for a complete description of each field in this struct
+    pub max_connections: usize,
+    pub max_fragment_size: usize,
+    pub max_in_buffer_capacity: usize,
+    pub max_out_buffer_capacity: usize,
+}
+
+impl Default for PubSubConfig {
+    fn default() -> Self {
+        Self {
+            max_connections: 1000,             // Arbitrary, default of 100 is too low
+            max_fragment_size: 50 * 1024,      // 50KB
+            max_in_buffer_capacity: 50 * 1024, // 50KB
+            max_out_buffer_capacity: 10 * 1024 * 1024 + 1024, // max account size (10MB) + extra (1K)
```

Comment:
I think we'll need closer to 14MB here to cover the base64 encoding overhead

Reply:
I'm a little worried about affecting the current `max_payload` default, which is 10MB

Reply:
I'll bump it to 15MB on master/v1.3.   But for the v1.2 backport, I'll keep the original 10MB value

Reply:
Sounds like a plan!